zipkin-lens/scss/main.scss

```diff
@@ -39,16 +39,15 @@ $fa-font-path: "~@fortawesome/fontawesome-free/webfonts";
 @import 'components/browser';
 @import 'components/browser-header';
 @import 'components/browser-results';
-@import 'components/condition-duration';
-@import 'components/condition-lookback';
-@import 'components/condition-name';
-@import 'components/condition-tags';
```

Comment:
We don't need these css files anymore, because material-ui's style-system is more excellent than this way of importing SCSS files.
Please see: https://material-ui.com/styles/basics/